ppr-ui/src/main.ts

```diff
@@ -58,9 +58,23 @@ async function start () {
     await initLdClient()
   }
 
-  // configure KeyCloak Service
-  console.info('Starting Keycloak service...') // eslint-disable-line no-console
-  await KeycloakService.setKeycloakConfigUrl(sessionStorage.getItem('KEYCLOAK_CONFIG_PATH'))
+  // Initialize Keycloak / sync SSO
+  await syncSession()
+
+  async function syncSession () {
```

Comment:
This is currently a nested function, which is legal but usually makes the parent function pretty big.

I suggest either:
- move this function to the bottom of this file
- change this function to inline code

Reply:
Thanks for the fix.

Reply:
(I do not have permissions to resolve this conversation.)